src/Components/login.css

```diff
@@ -0,0 +1,33 @@
+.login-main {
+    height: 100vh;
+    background-color: #eaeaea;
+  }
+  
+  .login-content {
+    left: 50%;
+    padding: 10vmin;
+    position: fixed;
+    text-align: center;
+    top: 50%;
+    transform: translate(-50%, -50%);
+  }
+  
+  .login-card {
+    background-color: #ffffff;
+    box-shadow: 3px 3px 8px #888888;
+    border-radius: 4px;
+    width: 400px;
+    min-width: 300px;
+    text-align: center;
+  }
+  .login-logo {
+    width: 118px;
+    margin-bottom: -10px;
+  }
+  .login-form {
+    margin-top: 35px;
+  }
+  .login-input-field {
+    font-size: 18px;
+    margin-bottom: 25px;
+  }
```

Comment:
This `.css` file should not live with the components. Consider moving it to either a new `style` directory or move it to `index.css`

Reply:
In the interest of time, we will create a chore and sort this out at a later stage.